manifests/Docker/DockerDesktopEdge/2.3.1.0.45408.yaml

```diff
@@ -0,0 +1,25 @@
+# ================== General ==================
+
+Id: Docker.DockerDesktopEdge
+Publisher: Docker Inc.
+Name: Docker Desktop Edge
+AppMoniker: docker
```

Comment:
This should be removed I guess, we now have two monikers.

Reply:
https://github.com/microsoft/winget-pkgs/pull/1390